src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -77,7 +78,7 @@ public class EvtItem extends SkriptEvent {
 			Skript.registerEvent("Prepare Craft", EvtItem.class, PrepareItemCraftEvent.class, "[player] (preparing|beginning) craft[ing] [[of] %-itemtypes%]")
 					.description("Called just before displaying crafting result to player. Note that setting the result item might or might not work due to Bukkit bugs.")
 					.examples("on preparing craft of torch:")
-					.since("2.2-Fixes-V10");
+					.since("2.2-Fixes-V10)");
```

Comment:
Why is there a closing bracket inside the string with no opening bracket?

Reply:
No idea how it got there. Thanks for the heads up!